package.json

```diff
@@ -2,19 +2,15 @@
   "name": "qiskit_sphinx_theme",
   "version": "1.0.0",
   "devDependencies": {
-    "jest": "^29.5.0"
+    "http-server": "^14.1.1",
+    "@playwright/test": "^1.34.3",
+    "playwright": "^1.34.3"
   },
   "scripts": {
-    "test": "jest"
-  },
-  "jest": {
-    "testPathIgnorePatterns": [
-      "/node_modules/",
-      "/.venv",
-      "/venv",
-      "/.tox",
-      "/build",
-      "/dist"
-    ]
+    "start": "cd example_docs/docs/_build/html; http-server",
+    "test": "npm run _docker-build && npm run _docker-run",
```

Comment:
I think for npm commands we should be more explicit that this is for testing snapshots, so change to "test-snapshot" or similar